tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -96,8 +96,16 @@ def test_backward_gpu(self):
 
     def check_double_backward(self, x_data, t_data, y_grad, x_grad_grad,
                               t_grad_grad):
+
+        delta = 1
+        eps = self.double_backward_options['eps']
+        xp = cuda.cupy.get_array_module(x_data)
```

Comment:
Could you fix the error?
`AttributeError: 'object' object has no attribute 'get_array_module'`

Reply:
Concretely,
```suggestion
        xp = chainer.get_array_module(x_data)
```
or use `abs` instead of `xp.abs`.